modules/making-disruptive-changes-br-ex-bridge.adoc

```diff
@@ -0,0 +1,122 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_bare_metal/bare-metal-postinstallation-configuration.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="making-disruptive-changes-br-ex-bridge.adoc_{context}"]
+= Making disruptive changes to a customized `br-ex` bridge
+
+For certain situations, you might need to make disruptive changes to a `br-ex` bridge for planned maintenance or network configuration updates. A `br-ex` bridge is a gateway for all external network traffic from your workloads, so any change to the bridge might temporarily disconnect pods and virtual machines (VMs) from an external network.
+
+The following procedure uses an example to show making disruptive changes to a `br-ex` bridge that minimizes any impact to running cluster workloads. 
+
+For all the nodes in your cluster to receive the `br-ex` bridge changes, you must reboot your cluster. Editing the existing `MachineConfig` object does not force a reboot operation, so you must create an additional `MachineConfig` object to force a reboot operation for the cluster.
+
+[IMPORTANT]
+====
+As a postinstallation task, you cannot change the `ipv4.address.ip` or the `ipv6.address.ip` parameter values for an existing `MachineConfig` object. Changing the IP addresses for nodes can disrupt critical networking routes, causing pods to lose connectivity.
```

Comment:
Changing Node IP is tricky, because it is/was an implicit requirement due to Reverse DNS and TLS certificate changes.   If/when we do change the requirement and allow for Node IP changes, we will have to track down all these statements. 

It would be nice to cross-reference this to somewhere official in the docs, but I don't think we have the requirement documented.